File: section3/26/ex.go
package main

import "fmt"

func findNemo(array []string) {
	for i := 0; i < len(array); i++ {
		if (array[i] == "nemo") {
			fmt.Print("Found nemo")
		}
	}
}

func main () {
	nemo := []string{"nemo"};
	findNemo(nemo)
}

File: section3/27BigO_scalability/ex.js
const {performance} = require('perf_hooks');

const everyone = ['dory','bruce','martin','nemo','gill','bloat','nigel','squirt','darla','hank'];

function findNemo (array) {
    // let t0 = performance.now()
    for (let i=0; i<array.length; i++) {
        if(array[i] === 'nemo'){
            console.log('Found nemo');
            // break;
        }
    }
    // let t1 = performance.now()
    // console.log(t1-t0)
}

findNemo(everyone);